gatsby-node.js

```diff
@@ -57,6 +87,16 @@ exports.createPages = async ({graphql, boundActionCreators}) => {
     throw Error(allMarkdown.errors);
   }
 
+  const issues = detectDuplicateRedirectFroms(allMarkdown.data.allMarkdownRemark.edges);
+  if (issues.length) {
+    console.error('The following paths are being redirected to different permalinks:\n' +
+      issues.map(issue => `  - ${issue.fromPath} → ${issue.permalinks.join(', ')}`).join('\n')
+    );
+    throw new Error('Found the following redirect_froms to be duplicated:\n' +
```

Comment:
Might have to change this to `process.exit(1)` if we want the build to halt.

Reply:
Agreed. Replacing the `throw` with `process.exit(1)` seems like the right thing to do.